openapi/multi-yaml/components/schemas/TerminalsOrder.yaml

```diff
@@ -24,6 +24,63 @@ properties:
     type: string
     description: business legal name when the terminal order was created
     example: Business Name
+  mcc:
+    type: string
+    description: Merchant Category Code
+    example: 7998
+  receiver_name:
+    type: string
+    description: name of the person receiving the terminal
+    example: John Doe
+  contact_first_name:
+    type: string
+    description: company's representative first name
+    example: John
+  contact_last_name:
+    type: string
+    description: company's representative last name
+    example: Doe
+  contact_email:
+    type: string
+    description: company's contact e-mail
+    example: john.doe@example.com
+  contact_phone_number:
+    type: string
+    description: company's contact phone number
+    example: 2125554567
+  line1:
+    type: string
+    example: 123 Main St
+  line2:
+    type: string
+    example: Apt 4B
+  city:
+    type: string
+    example: Minneapolis
+  state:
+    type: string
+    example: MN
+  postal_code:
+    type: string
+    example: 55401
+  time_zone:
+    type: string
+    description: determined by postal code
+    example: US/Central
+  country:
+    type: string
+    example: USA
+  shipping_tracking_reference:
+    type: string
+    example: 12345678
```

Comment:
Could you please add a description here and mentions that this is a FedEx tracking number and that it is only available once the terminal order status is completed - and always null for boarding_only orders?